config/manifests/bases/kepler-operator.clusterserviceversion.yaml

```diff
@@ -71,4 +350,4 @@ spec:
   provider:
     name: https://sustainable-computing.io/
```

Comment:
Name is string. Use "Kepler Operator Contributors" That is why the link is not rendering.